### DISCUSSION TITLE
Informações de Contato

### DISCUSSION THREAD

**Discussion by ramonsilva-rd** [RamonSilva20/mapos#2059] [General]
Bom dia pessoal, 

Meu chamo Ramon Silva, também sou programado mas  não sou o mesmo Ramon que mantem esse projeto. Ultimamente tenho recebido dezenas de emails de pessoas me perguntando sobre o sistema e pedindo ajuda, dentro do possível eu redireciona-os para esta página. Então eu gostaria de sugerir a vocês deixarem uma opção bem visível de contato, pois eles sempre procuram no google, e meu site vem em primeiro no resultado ai me mandam e-mail, mensagens e até ligam pro meu telefone. 

Desde de já agradeço

**Comment by willph** [comment]
Tem uma comunidade onde os usuários se ajudam entre se. 

https://chat.whatsapp.com/D571nPLqNq6JgIsmL6Gucl

Ou email

contato@mapos.com.br
